user/templates/debian/debian.md

```diff
@@ -11,11 +11,20 @@ ref: 134
 title: Debian templates
 ---
 
-The Debian [template](/doc/templates/) is an officially [supported](/doc/supported-releases/#templates) template in Qubes OS.
-This page is about the standard (or "full") Debian template.
-For the minimal version, please see the [Minimal templates](/doc/templates/minimal/) page.
+The Debian [template](/doc/templates/) is an officially [supported](/doc/supported-releases/#templates) template in Qubes OS.  
+This page is about the standard (or "Full") Debian template.
+For the Minimal version, please see the [Minimal templates](/doc/templates/minimal/) page.
```

Comment:
I don't think these terms should be capitalized.

Reply:
Cool!